bin/adjust.ts

```diff
@@ -44,18 +49,37 @@ async function task(file: string) {
       astray.walk<MDAST.Heading, void, any>(node, {
         text(t: MDAST.Text) {
           title += t.value;
-        }
+        },
       });
 
       return astray.SKIP;
-    }
+    },
+
+    link(node: MDAST.Link) {
+      if (node.url.startsWith('/')) {
+        node.url = `/${product}${node.url}`;
+      }
+    },
+
+    html(node: MDAST.HTML) {
+      if (node.value.startsWith('<button ')) {
+        let html = parse(node.value);
+        let button = html.querySelector('button');
+        let href = button?.getAttribute('href');
+        if (button && href && href.startsWith('/')) {
+          button.setAttribute('href', `/${product}${href}`);
+          node.value = button.toString().replace('</button>', '');
```

Comment:
lol I don't even know.. this is pretty bad but I can't find a parser that lets you specify "no closing tag".. honestly this whole HTML parser can go if you don't like it :)